my-app/src/NavBar.js

```diff
@@ -2,14 +2,20 @@
 import logo from './logo.svg';
```

Comment:
added a logout button to front page but needs to be properly formatted - currently sits in the middle of the navbar...